website/database.py

```diff
@@ -242,14 +242,6 @@ def get_hedy_choices(self):
     def set_program_as_hedy_choice(self, id, favourite):
         PROGRAMS.update({'id': id}, {'hedy_choice': 1 if favourite else None})
 
-    def all_programs_count(self):
```

Comment:
why are we removing these two functions?

Reply:
They are no longer used, the user count is already calculated from the length of the users list; reducing the amount of db calls. The programs counts function has never been used in the first place.

Reply:
ok!